homework8/task1.py

```diff
@@ -0,0 +1,16 @@
+from datetime import datetime
+
+
+def decorator(func):
+  def wrap():
```

Comment:
Use *args and **kwargs here to make in works with functions that take arguments. And pass these arguments to the func() call